In [13]:
import pandas as pd
import matplotlib.pyplot as plt
import numpy as np
import seaborn as sns

data = pd.read_csv('../../data/events.csv', low_memory=False)

data['fecha'] = pd.to_datetime(data['timestamp'], errors='coerce')
data['mes'] = data['fecha'].dt.month
data['dayofyear'] = data['fecha'].dt.dayofyear
data['semana'] = data['fecha'].dt.week

sns.set(style="white", context="talk")

In [5]:
data.staticpage.value_counts()

CustomerService                      1528
AboutUs                               443
FaqEcommerce                          347
Quiosks                               326
trust-trocafone                       243
galaxy-s8                             167
TermsAndConditionsReturnEcommerce     156
how-to-sell                            96
Conditions                             89
how-to-buy                             86
TermsAndConditionsEcommerce            59
club-trocafone                         46
PrivacyEcommerce                        9
black_friday                            3
Name: staticpage, dtype: int64

In [48]:
static = data[data.staticpage == 'galaxy-s8']
static = static.groupby('dayofyear').staticpage.value_counts().to_frame('visitas')
static = static.unstack('staticpage')
static
# static.fillna(0).plot.line(figsize = (20, 10))

visitas
staticpage galaxy-s8
dayofyear           
113               27
114               45
115               41
116               31
117                2
119                1
120                3
123                1
125                2
128                1
129                3
133                2
137                2
138                2
140                1
141                3

En el día 114 se produce un incremento de visitas hacia la página 'Galaxy-S8'. ¿Podría ser que se haya incrementado la publicidad a causa de la publicación de este telefono y eso haya incrementado las visitas?